### PR TITLE
convert tex formulas in clmm/modeling.py

### PR DESCRIPTION
**Pull request by m-aguena**

### CODE REVIEW

File: clmm/modeling.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def get_3d_density(r3d, mdelta, cdelta, cosmo, Delta=200, halo_profile_parameter
         Mass overdensity definition; defaults to 200.
     halo_profile_parameterization : str, optional
         Profile model parameterization, with the following supported options:
-        `nfw` (default) - $\rho(r) = \frac{\rho_0}{c/(r/R_{vir})(1+c/(r/R_{vir}))^2}$ [insert citation here]
+        `nfw` (default) - :math:`\rho(r) = \frac{\rho_0}{c/(r/R_{vir})(1+c/(r/R_{vir}))^2}` [insert citation here]
```
Comment (matthewkirby) [inline clmm/modeling.py]: Let's change this to give options here but descriptions of the options in the main section of the docstring above. This is too much for a parameter description.

File: clmm/modeling.py [hunk 2]
```diff
@@ -344,7 +344,7 @@ def predict_tangential_shear(r_proj, mdelta, cdelta, z_cluster, z_source, cosmo,
 
 def predict_convergence(r_proj, mdelta, cdelta, z_cluster, z_source, cosmo, Delta=200,
                         halo_profile_parameterization='nfw', z_src_model='single_plane'):
-    r"""Computes the mass convergence $\kappa = \frac{\Sigma}{\Sigma_{crit}}$ or $\kappa = \kappa_\inf \times \Beta_s$
+    r"""Computes the mass convergence :math:`\kappa = \frac{\Sigma}{\Sigma_{crit}}` or :math:`\kappa = \kappa_\inf \times \Beta_s`
```
Comment (matthewkirby) [inline clmm/modeling.py]: \beta not \Beta

File: clmm/modeling.py [hunk 3]
```diff
@@ -168,7 +168,7 @@ def predict_surface_density(r_proj, mdelta, cdelta, cosmo, Delta=200, halo_profi
 def predict_excess_surface_density(r_proj, mdelta, cdelta, cosmo, Delta=200, halo_profile_parameterization='nfw'):
     r"""Computes the excess surface density
 
-    $\Delta\Sigma(R) = \bar{\Sigma}(<R)-\Sigma(R)$, where $\bar{\Sigma}(<R) = \frac{2}{R^2} \int^R_0 dR' R' \Sigma(R')$
+    :math:`\Delta\Sigma(R) = \bar{\Sigma}(<R)-\Sigma(R)`, where :math:`\bar{\Sigma}(<R) = \frac{2}{R^2} \int^R_0 dR' R' \Sigma(R')`
```
Comment (matthewkirby) [inline clmm/modeling.py]: This shouldn't be inline math.

Inline math ``:math:`equation` ``

Non-Inline math: 
```
..math::
    equation

more text
```
Note that the equation should be tabbed in and there needs to be an after the equation (maybe also before, I don't remember. Compile the docs and take a look)

File: clmm/modeling.py (hunk 2 above)
Comment (matthewkirby) [inline clmm/modeling.py]: Lets make this non-inline math as well

File: clmm/modeling.py [hunk 4]
```diff
@@ -250,7 +250,7 @@ def get_angular_diameter_distance_a(cosmo, aexp2, aexp1=1.):
 def get_critical_surface_density(cosmo, z_cluster, z_source):
     r"""Computes the critical surface density
 
-    $\Sigma_{crit} = \frac{c^2}{4\pi G} \frac{D_s}{D_LD_{LS}}$
+    :math:`\Sigma_{crit} = \frac{c^2}{4\pi G} \frac{D_s}{D_LD_{LS}}`
```
Comment (matthewkirby) [inline clmm/modeling.py]: Non-inline math this

File: clmm/modeling.py [hunk 5]
```diff
@@ -148,7 +148,7 @@ def predict_surface_density(r_proj, mdelta, cdelta, cosmo, Delta=200, halo_profi
     Returns
     -------
     sigma : array_like, float
-        Surface density, Sigma in units of [h Msun/$pc^2$]
+        Surface density, Sigma in units of [:math:`h M_odot/pc^2`]
```
Comment (matthewkirby) [inline clmm/modeling.py]: \odot

File: clmm/modeling.py [hunk 6]
```diff
@@ -122,7 +122,7 @@ def get_3d_density(r3d, mdelta, cdelta, cosmo, Delta=200, halo_profile_parameter
 def predict_surface_density(r_proj, mdelta, cdelta, cosmo, Delta=200, halo_profile_parameterization='nfw'):
     r"""Computes the surface mass density
 
-    $\Sigma(R) = \Omega_m \rho_{crit} \int^\inf_{-\inf} dz \Xi_{hm} (\sqrt{R^2+z^2})$, where $\Xi_{hm}$ is the halo mass function.
+    :math:`\Sigma(R) = \Omega_m \rho_{crit} \int^\inf_{-\inf} dz \Xi_{hm} (\sqrt{R^2+z^2})`, where :math:`\Xi_{hm}` is the halo mass function.
```
Comment (matthewkirby) [inline clmm/modeling.py]: non-inline equation

Comment (matthewkirby) [inline clmm/modeling.py]: \infty

File: clmm/modeling.py [hunk 7]
```diff
@@ -194,7 +194,7 @@ def predict_excess_surface_density(r_proj, mdelta, cdelta, cosmo, Delta=200, hal
     Returns
     -------
     deltasigma : array_like, float
-        Excess surface density, DeltaSigma in units of [h Msun/$pc^2$].
+        Excess surface density, DeltaSigma in units of [:math:`h M_odot/pc^2`].
```
Comment (matthewkirby) [inline clmm/modeling.py]: \odot

File: clmm/modeling.py [hunk 8]
```diff
@@ -288,9 +288,9 @@ def predict_tangential_shear(r_proj, mdelta, cdelta, z_cluster, z_source, cosmo,
                              halo_profile_parameterization='nfw', z_src_model='single_plane'):
     r"""Computes the tangential shear
 
-    $\gamma_t = \frac{\Delta\Sigma}{\Sigma_{crit}} = \frac{\bar{\Sigma}-\Sigma}{\Sigma_{crit}}}$
+    :math:`\gamma_t = \frac{\Delta\Sigma}{\Sigma_{crit}} = \frac{\bar{\Sigma}-\Sigma}{\Sigma_{crit}}}`
```
Comment (matthewkirby) [inline clmm/modeling.py]: This one didn't work at all (I think that theres an extra `}`?).

Comment (matthewkirby) [inline clmm/modeling.py]: Non-inline

File: clmm/modeling.py [hunk 9]
```diff
@@ -288,9 +288,9 @@ def predict_tangential_shear(r_proj, mdelta, cdelta, z_cluster, z_source, cosmo,
...
     or
-    $\gamma_t = \gamma_\inf \times \Beta_s$
+    :math:`\gamma_t = \gamma_\inf \times \Beta_s`
```
Comment (matthewkirby) [inline clmm/modeling.py]: \beta not \Beta

File: clmm/modeling.py [hunk 10]
```diff
@@ -397,7 +397,7 @@ def predict_convergence(r_proj, mdelta, cdelta, z_cluster, z_source, cosmo, Delt
 
 def predict_reduced_tangential_shear(r_proj, mdelta, cdelta, z_cluster, z_source, cosmo, Delta=200,
                                      halo_profile_parameterization='nfw', z_src_model='single_plane'):
-    r"""Computes the reduced tangential shear $g_t = \frac{\\gamma_t}{1-\\kappa}$.
+    r"""Computes the reduced tangential shear :math:`g_t = \frac{\\gamma_t}{1-\\kappa}`.
```
Comment (matthewkirby) [inline clmm/modeling.py]: Non-inline

File: clmm/modeling.py (hunk 1 above)
Comment (m-aguena) [reply clmm/modeling.py]: How about now?

File: clmm/modeling.py (hunk 2 above)
Comment (m-aguena) [reply clmm/modeling.py]: done

File: clmm/modeling.py (hunk 3 above)
Comment (m-aguena) [reply clmm/modeling.py]: must have an empty line and an extra space before `math`, does not work without it:
```
text

.. math::
    equation

more text
```

**Comment by m-aguena** [comment]
@matthewkirby  updated the docstrings and compiled them now with sphinx. Please take a look. Also, regarding one of the proposed items (Un latex-ed equations:￼ d_LS in  `get_angular_diameter_distance`), there is no `get_angular_diameter_distance` and `get_angular_diameter_distance_a` has no equation.

**Comment by matthewkirby** [comment]
Looks much better! Regarding the d_LS comment, its nbd since its a temporary function, it should be fine as it is. I am going to go ahead and merge this!

**Pull request merged**